devicetypes/qubino/qubino-flush-2-relay.src/qubino-flush-2-relay.groovy

```diff
@@ -114,6 +116,12 @@ def updated() {
 	// Preferences template end
 }
 
+private getEnergyUsage() {
+	for (def endpoint : 1..state.numberOfSwitches) {
+		sendHubCommand(encap(zwave.meterV3.meterGet(scale: 0x00)), endpoint)
```

Comment:
I might group all of these into a list first and then just pass that list to sendHubCommand